CMakeLists.txt

```diff
@@ -328,7 +328,11 @@ if((CLANG_ASAN_UBSAN OR CLANG_MSAN OR CLANG_TSAN) AND NOT CMAKE_C_COMPILER_ID MA
   message(FATAL_ERROR "Sanitizers are only supported for Clang.")
 endif()
 
-option(ENABLE_JEMALLOC "enable jemalloc" ON)
+if(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+  option(ENABLE_JEMALLOC "enable jemalloc" OFF)
+else()
+  option(ENABLE_JEMALLOC "enable jemalloc" ON)
+endif()
```

Comment:
You mean ignoring ENABLE_JEMALLOC entirely in openbsd,? I would not oppose someone fixing and testing the underlying issue with jemalloc in OpenBSD, i.e. ENABLE_JEMALLOC should work as intended even if it breaks things, its just not a sane default in OpenBSD.

But either way is fine with me, provided the default build instructions lead to a working build :D